app/Jobs/Schedule/RunTaskJob.php

```diff
@@ -46,8 +52,8 @@ public function handle(
         InitiateBackupService $backupService,
         DaemonPowerRepository $powerRepository
     ) {
-        // Do not process a task that is not set to active.
-        if (!$this->task->schedule->is_active) {
+        // Do not process a task that is not set to active, unless it's been trigger by the run now API.
+        if ($this->task->schedule->is_active && !$this->now) {
```

Comment:
I do not follow the reason for changing the logic right here. Shouldn't this be asking if the schedule is not active and `$this->now` is not set?

Otherwise it looks like this is just going to explode when a task runs because it is set to active but `$this->now` is false for those tasks?

Reply:
there's an accidental '!' removal :P This is fixed